### ISSUE TITLE
theme.gemspec doesn't build theme gem properly 

### ISSUE THREAD

**Issue by ashmaroli** [jekyll/jekyll#5323]
- [x] I believe this to be a bug, not a question about using Jekyll.
- [x] I updated to the latest Jekyll (or) if on GitHub Pages to the latest `github-pages`
- [x] I read the CONTRIBUTION file at https://jekyllrb.com/docs/contributing/
- [x] This is **not** a feature request.

---

- [x] I am on (or have tested on) ***Windows 7***

---

- [x] I was trying to build the theme gem.

## My Reproduction Steps

- Create new theme, **titan**:
```
$ jekyll new-theme titan
```
- cd into theme dir and add a file each to empty subdirs.
- edit *titan.gemspec* to prepare for building gem
- update git index by running:
```
$ git add .
```
- Build gem.
```sh
$ gem build titan.gemspec

  Successfully built RubyGem
  Name: titan
  Version: 0.1.0
  File: titan-0.1.0.gem
```
- unpack gem contents to verify build before publishing:
```sh
$ gem unpack titan-0.1.0.gem
Unpacked gem: ~/titan-0.1.0
```
- `cd` into the unpacked dir and inspect.

## The Output I Wanted

see the necessary files in the unpacked directory.

## The Output I Got

an empty directory which means my theme gem was essentially empty.

---
I reached a solution by simply changing the `spec.files` line in the gemspec to:
```ruby
  spec.files  = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
```
and additionally adding a file each to empty dirs.

---
/cc @jekyll/ecosystem

**Comment by benbalter** [comment]
> I reached a solution by simply changing the spec.files line in the gemspec to:

@ashmaroli this is a great bug report, thank you.

Any interested in submitting a pull request to make that change?

/cc @jekyll/ecosystem

**Comment by ashmaroli** [comment]
@benbalter thanks :smiley:  I've already opened a PR at #5325.

**Comment by benbalter** [comment]
See an alternative approach for minima via https://github.com/jekyll/minima/pull/31 (and related issue https://github.com/jekyll/minima/issues/29.

**Comment by ashmaroli** [comment]
Fixed in #5325
